cmd/fs-v1-metadata.go

```diff
@@ -274,167 +261,134 @@ func newFSMetaV1() (fsMeta fsMetaV1) {
 	return fsMeta
 }
 
-// newFSFormatV2 - initializes new formatConfigV1 with FS format version 2.
-func newFSFormatV2() (format *formatConfigV1) {
+// newFSFormatV1 - initializes new formatConfigV1 with FS format info.
+func newFSFormatV1() (format *formatConfigV1) {
 	return &formatConfigV1{
-		Version: "1",
-		Format:  "fs",
+		Version: formatFileCurrent,
```

Comment:
As we create `formatConfigV1`, `Version` should be `formatFileV1`.  `formatFileCurrent` would point to different version in future.